megamek/src/megamek/common/net/IConnection.java

```diff
@@ -16,33 +16,33 @@
 
 /**
  * interface for connections between client and server refactored from the
- * original Connection class which was moved to AbstractConnection
+ * original Connection class which was moved to AbstractSocketConnection
  */
 public interface IConnection {
     /**
      * Opens the connection
      * 
      * @return <code>true</code> on success, <code>false</code> otherwise
      */
-    public boolean open();
+    boolean open();
```

Comment:
IntelliJ (correctly) pointed out that for interfaces, there is no need to mark the methods as public (they are inherently public)